readme.md

```diff
@@ -559,7 +559,57 @@ const useStore = create(
   ),
 )
 ```
+  
+## Splitting the store into separate files and combining it 
+  
+When your project is big with different kind of datas, for convenient coding you want to keep small files. 
+So, like combineReducers in Redux, you can split your store in different functions that will be able to access each other, keeping track of global state. 
 
+
+```jsx
+
+import create from 'zustand'
+
+const bearStore = (set, get) => ({
```

Comment:
Technically, this is not a store, but a store creator. So, I would name it something like `createBearStore`.

Reply:
Yes it is just a slice, i will rename it : createBearStore or bearSlice ?

Reply:
I would prefer having `create` or `creator`.

Although it's a slice, it can be used alone.
```js
const createBearSlice = (set, get) => ...
const useStore = create(createBearSlice)
```
Now, create, create sounds weird in this case. So, maybe we don't need `create`. 🤔

Reply:
I see no problem, I renamed it createBearSlice.